tests/NuGetUtility.Test/ReferencedPackagesReader/ReferencedPackageReaderTest.cs

```diff
@@ -239,6 +239,25 @@ public void GetInstalledPackages_Should_ReturnCorrectValues_If_IncludingTransiti
                         new PackageIdentity(l.Object.Name, l.Object.Version)),
                 result);
         }
+        
+        [Test]
```

Comment:
Please add additional Unit Tests for these scenarios:
- Wildcard matches start instead of end
- Wildcard matches middle
- Include multiple wildcards that should match

Reply:
done